chore(docs) minor refresh on math skills test page

bsmth opened the pull request:
This PR does a quick pass over the math skills test page.

The original issue is resolved, in my opinion, it's clear that this page references the reading material needed to complete the assessment. In this case, I'm doing some minor rewording to simplify.

Fixes #11058

bsmth committed:
chore(docs) minor refresh on math skills test page

Josh-Cena commented on files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:
```diff
@@ -11,25 +11,27 @@ tags:
-> If you get stuck, then ask us for help — see the [Assessment or further help](#assessment_or_further_help) section at the bottom of this page.
-
-> **Note:** In the examples below, if there is an error in your code it will be outputted into the results panel on the page, to help you try to figure out the answer (or into the browser's JavaScript console, in the case of the downloadable version).
+> **Note:** If there is an error in your code it will be logged into the results panel on this page (or in the browser's JavaScript console in the downloadable version) to help you.
```
```suggestion
> **Note:** If there is an error in your code, it will be logged into the results panel on this page (or in the browser's JavaScript console for the downloadable version) to help you.
```

Josh-Cena commented on files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:
```diff
@@ -11,25 +11,27 @@ tags:
 
-Let's start out by testing your knowledge of basic math operators. You will have to create four numeric values, then add the first two together, then subtract the fourth from the third, then multiply the two secondary results together to get a result of 48. Finally, we need to write a test that proves that this value is an even number.
+Let's start out by testing your knowledge of basic math operators.
+You will have to create four numeric values, adding two together, subtracting one from another, then multiplying the results.
```
```suggestion
You will create four numeric values, add two together, subtract one from another, then multiply the results.
```

Josh-Cena commented on files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:
```diff
@@ -11,25 +11,27 @@ tags:
+Try updating the live code below to recreate the finished example by following these steps:
 
-1. Create four variables that contain numbers. Call the variables something sensible.
+1. Create four variables with sensible names that contain numbers.
```
This is ambiguous and can mean "the names contain numbers". Original reads better.

bsmth replied:
Good point, have reverted

bsmth committed:
Apply suggestions from code review

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

bsmth commented on files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:
```diff
@@ -11,25 +11,27 @@ tags:
+Try updating the live code below to recreate the finished example by following these steps:
 
-1. Create four variables that contain numbers. Call the variables something sensible.
+1. Create four variables with sensible names that contain numbers.
```
```suggestion
1. Create four variables that contain numbers. Call the variables something sensible.
```

bsmth committed:
Update files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md

bsmth committed:
Merge branch 'main' into 11058-skills-math

bsmth committed:
Merge branch 'main' into 11058-skills-math

bsmth commented:
Thanks @Josh-Cena, was there anything else outstanding on this for me? I'm happy to pick it up again.

Josh-Cena commented on files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:
```diff
@@ -11,25 +11,27 @@ tags:
+4. Multiply the results from steps **2** and **3** and store the result in a variable called `finalResult`.
+5. Check whether `finalResult` is an even number and store the result in a variable called `evenOddResult`.
+
+To pass this test, you should have a result of `48`.
```
The "result" here is referring to `finalResult`, but that's slightly ambiguous. What about inlining it in step 4, or explicitly mention the values for both `finalResult` and `evenOddResult`?

bsmth replied:
Added some changes in 9e139acb1e20673a6ce3b5b4b78622c3dfa92087

Josh-Cena commented on files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:
```diff
@@ -39,16 +41,17 @@ So, try updating the live code below to recreate the finished example, following
 
-In the second task you are already provided with two calculations, with the results stored in the variables `result` and `result2`. But these results aren't what we want; you'll need to take the calculations and change them to give us what we want.
+In the second task, you are provided with two calculations with the results stored in the variables `result` and `result2`.
+You need to take the calculations and change their formatting so it has two decimal places.
```
```suggestion
You need to take the calculations, multiply them, and format the result to two decimal places.
```

...I think?

Josh-Cena reviewed (commented):
Some suggestions. I can't tell if the new tone is better for a new learner, but I'm neutral on this.

bsmth committed:
Update files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

bsmth committed:
Merge remote-tracking branch 'upstream/main' into 11058-skills-math

bsmth committed:
Add hints for arithmetic operators in JavaScript math section

bsmth committed:
Merge branch 'main' into 11058-skills-math

bsmth committed:
Merge branch 'main' into 11058-skills-math

bsmth commented:
Hi @Josh-Cena, I've addressed your review comments, thank you. Could you have another quick look?

Josh-Cena reviewed (approved):
Sorry @bsmth I keep forgetting about this. This LGTM, thanks!

I'm leaving it open though, because frankly I don't understand what https://github.com/mdn/content/issues/11058 is asking for. If you believe it adequately addresses that issue (or if that issue needs to be addressed at all), feel free to self-merge. Otherwise some more context would be helpful.

bsmth commented:
Not a problem, thank you!

> If you believe it adequately addresses that issue (or if that issue needs to be addressed at all), feel free to self-merge. Otherwise some more context would be helpful.

I believe this page addresses the concerns in the issue, specifically this:

>lack of reference to the information needed to complete the task being asked of the learner

I think it's clear from the material that readers should be applying skills learned in the previous section, so this is good to go, in my opinion.

Thanks for having a look, I will merge shortly 👍🏻

bsmth committed:
Merge branch 'main' into 11058-skills-math
